modules/ROOT/pages/mutations/delete.adoc

```diff
@@ -21,7 +21,7 @@ type User {
 }
 ----
 
-These `delete` mutations and response types should be generated:
+The following `delete` mutations and response types are generated:
```

Comment:
Is there a reason why the mutation is here written in lowercase? It could be helpful to find a rule for that, but I think generally we write clauses/operations in uppercase.

Reply:
hm... on second thought, not so sure actually
in code listings, it's a prefix and then it's used as lower case (`deletePosts` for `Post` objects)

for mutations, it's used consistently as lower case actually (create, delete and update are all written like this)

Reply:
Ah, whatever works best! I was just wondering because there are some cases where they are written in uppercase.